docs/pages/core-concepts.mdx

```diff
@@ -0,0 +1,224 @@
+
+## Teleport editions
+
+Teleport is available in three **editions**. All editions include the same open
+source core, which is available at the
+[`gravitational/teleport`](https://github.com/gravitational/teleport) repository
+on GitHub.
+
+You can find a detailed comparison of Teleport's editions in our [Frequently
+Asked Questions](./faq.mdx#how-is-open-source-different-from-enterprise) page.
+
+### Teleport Community Edition
+
+**Teleport Community Edition** is a free, open source distribution of Teleport
+that anyone can download, install, and host on their own infrastructure.
+
+### Teleport Enterprise
+
+**Teleport Enterprise** is a paid plan that includes all of the features of
+**Teleport Community Edition**, plus advanced features for organizations with
+advanced security needs, such as support for Federal Information Processing
+Standards (FIPS) and a hardware security module (HSM). Teleport Enterprise
+includes a support agreement with Teleport.
+
+Read more in our [Teleport Enterprise
+guide](./deploy-a-cluster/teleport-enterprise/introduction.mdx).
+
+### Teleport Enterprise Cloud
+
+**Teleport Enterprise Cloud** is a managed deployment of the **Teleport Auth
+Service** and **Teleport Proxy Service**.
+
+Our team at Teleport handles all tasks related to running the **Auth Service**
+and **Proxy Service**, including upgrades and certificate management. Each
+customer account, known as a **Teleport Enterprise Cloud tenant**, has its own
+subdomain of `.teleport.sh`, e.g., `mytenant.teleport.sh`.
+
+Read more in our [Teleport Enterprise Cloud
+guide](./deploy-a-cluster/teleport-cloud/introduction.mdx).
+
+## Configuration resources
+
+A **configuration resource** is a document stored on the **Teleport Auth
+Service** backend that specifies settings for your **Teleport cluster**.
+Examples include **roles**, **local users**, and **authentication connectors**
+
+Read more in our [resource reference](./reference/resources.mdx).
+
+### Role
+
+A **role** is a **configuration resource** that grants **Teleport users**
+privileges within a cluster. Teleport's role-based access control (RBAC) is
+restrictive by default, and a user needs explicit permissions before they can
+access a resource or perform management tasks on a cluster.
+
+Read our [guide to Teleport roles](./access-controls/guides/role-templates.mdx).
+
+### Teleport users
+
+Teleport allows for three kinds of users:
+
+- **Local users** correspond to a `user` **configuration resource** stored on
+  the **Auth Service** backend.
+- **Single Sign-On (SSO) users** are stored on the backend of your SSO solution,
+  e.g., Okta or GitHub. When a user authenticates to Teleport via your SSO
+  solution, Teleport issues a certificate for the user and creates a temporary
+  **local user** that is valid for the lifetime of the certificate.
+- **Bot users** are automated services that receive credentials via **Machine ID**.
```

Comment:
Technically there isn't actually a resource that is called bot user. They are the same as local users, they just don't have passwords.